packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1118,6 +1125,58 @@ export class StatsAnalyzer extends EventsScope {
     }
   }
 
+  /**
+   * Processes remote and local candidate result and stores
```

Comment:
this comment doesn't match what the function is doing

Reply:
fixed. ty.